src/windows_registry.rs

```diff
@@ -451,9 +477,9 @@ mod impl_ {
     }
 
     fn vs15plus_vc_paths(
-        target: &str,
+        target: TargetArch<'_>,
         instance_path: &Path,
-    ) -> Option<(PathBuf, PathBuf, PathBuf, PathBuf, PathBuf)> {
+    ) -> Option<(PathBuf, PathBuf, PathBuf, PathBuf, Option<PathBuf>, PathBuf)> {
```

Comment:
I think this return type is getting increasingly out of hand and should probably be a proper struct or otherwise rearchitectured. However, `vs15plus_vc_paths` is only called from one place and that immediately deconstructs it into named variables so I'm not minded to block this PR on that.